content/en/docs/tasks/job/parallel-processing-expansion.md

```diff
@@ -90,6 +102,9 @@ We can check on the pods as well using the same label selector:
 
 ```shell
 kubectl get pods -l jobgroup=jobexample
+```
+
```

Comment:
please add The output is similar to this: as per the guidelines before any output